<launch>
    <node name="left_wheel_pid" pkg="pid" type="controller" output="screen" >
      <param name="Kp" value="3.9" />
      <param name="Ki" value="5.9" />
      <param name="Kd" value="7.9" />
      <param name="rate" value="20" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="10" />
      <param name="cutoff_frequency" value="20" />
      <param name="topic_from_controller" value="control_effort" />
      <param name="topic_from_plant" value="state" />
      <param name="node_name" value="left_wheel" />
    </node>
    <node name="servo_sim_node" pkg="pid" type="plant_sim" output="screen" args="2" />
    <node name="setpoint_node" pkg="pid" type="setpoint_node" output="screen" />
    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />
    <node name="rqt_robot_monitor" pkg="rqt_robot_monitor" type="rqt_robot_monitor" />
    <node name="rqt_plot" pkg="rqt_plot" type="rqt_plot" />
</launch>
